econml/iv/dr/_dr.py

```diff
@@ -755,19 +2210,15 @@ def fit(self, Y, T, Z, X=None, W=None, *, sample_weight=None, groups=None,
         """
         # TODO: do correct adjustment for sample_var
         return super().fit(Y, T, Z=Z, X=X, W=W,
-                           sample_weight=sample_weight, groups=groups,
+                           sample_weight=sample_weight, freq_weight=freq_weight, sample_var=sample_var, groups=groups,
                            cache_values=cache_values, inference=inference)
 
-    def refit_final(self, *, inference='auto'):
-        return super().refit_final(inference=inference)
-    refit_final.__doc__ = _OrthoLearner.refit_final.__doc__
-
     @property
-    def bias_part_of_coef(self):
+    def fit_cate_intercept_(self):
         return self.ortho_learner_model_final_._fit_cate_intercept
 
     @property
-    def fit_cate_intercept_(self):
+    def bias_part_of_coef(self):
```

Comment:
IF we change tdesign: this method should not be implemented and the parent will return False by default